mtgjson5/classes/mtgjson_prices.py

```diff
@@ -12,18 +12,20 @@ class MtgjsonPricesObject:
     source: str
     provider: str
     date: str
+    currency: str
     buy_normal: Optional[float]
     buy_foil: Optional[float]
     sell_normal: Optional[float]
     sell_foil: Optional[float]
 
-    def __init__(self, source: str, provider: str, date: str) -> None:
+    def __init__(self, source: str, provider: str, date: str, currency: str) -> None:
```

Comment:
Just a thought: Could make an enum for pricing to make it easier to add new ones and not have so many rogue strings

class Currency(enum): USD = 1, EUR = 2